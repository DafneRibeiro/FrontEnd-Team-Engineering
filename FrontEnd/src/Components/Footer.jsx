const Footer = () => {
    return (
        <footer className="bg-light text-center text-lg-start ">
            <div className="text-center p-3" >
                <span className=" p-3">
                    Â© 2022 Copyright:
                    <a className='text-dark' href="https://digitalfutures.com/">Digital Futures</a>
                </span>
                <a className="p-1" href="https://www.linkedin.com/company/digital-futures2021/">
                    <i className="bi bi-linkedin"></i>
                </a>
                <a className="p-1" href="https://www.instagram.com/digital_futures/?hl=en">
                    <i className="bi bi-instagram"></i>
                </a>
                <a className="p-1" href="https://twitter.com/digitalfutures0">
                    <i className="bi bi-twitter"></i>
                </a>
                <a className="p-1" href="https://www.facebook.com/digital.futures2021/">
                    <i className=" bi bi-facebook"></i>
                </a>

            </div>
        </footer>
    )
}

export default Footer;