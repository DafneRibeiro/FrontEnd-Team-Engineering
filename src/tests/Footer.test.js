import { render, screen } from '@testing-library/react';
import Footer from '../Components/Footer';

test(`Footer matches snapshot`, () => {
    // render(<Footer />)
    // const copyright = screen.getByText(/copyright/i)
    // expect(copyright).toMatch('Copyright')
    expect(render(<Footer />)).toMatchSnapshot();

});