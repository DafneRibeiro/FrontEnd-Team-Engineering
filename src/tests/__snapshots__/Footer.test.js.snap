// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer matches snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <footer
        class="bg-light text-center text-lg-start "
      >
        <div
          class="text-center p-3"
        >
          <span
            class=" p-3"
          >
            © 2022 Copyright:
            <a
              class="text-dark"
              href="https://digitalfutures.com/"
            >
              Digital Futures
            </a>
          </span>
          <a
            class="p-1"
            href="https://www.linkedin.com/company/digital-futures2021/"
          >
            <i
              class="bi bi-linkedin"
            />
          </a>
          <a
            class="p-1"
            href="https://www.instagram.com/digital_futures/?hl=en"
          >
            <i
              class="bi bi-instagram"
            />
          </a>
          <a
            class="p-1"
            href="https://twitter.com/digitalfutures0"
          >
            <i
              class="bi bi-twitter"
            />
          </a>
          <a
            class="p-1"
            href="https://www.facebook.com/digital.futures2021/"
          >
            <i
              class=" bi bi-facebook"
            />
          </a>
        </div>
      </footer>
    </div>
  </body>,
  "container": <div>
    <footer
      class="bg-light text-center text-lg-start "
    >
      <div
        class="text-center p-3"
      >
        <span
          class=" p-3"
        >
          © 2022 Copyright:
          <a
            class="text-dark"
            href="https://digitalfutures.com/"
          >
            Digital Futures
          </a>
        </span>
        <a
          class="p-1"
          href="https://www.linkedin.com/company/digital-futures2021/"
        >
          <i
            class="bi bi-linkedin"
          />
        </a>
        <a
          class="p-1"
          href="https://www.instagram.com/digital_futures/?hl=en"
        >
          <i
            class="bi bi-instagram"
          />
        </a>
        <a
          class="p-1"
          href="https://twitter.com/digitalfutures0"
        >
          <i
            class="bi bi-twitter"
          />
        </a>
        <a
          class="p-1"
          href="https://www.facebook.com/digital.futures2021/"
        >
          <i
            class=" bi bi-facebook"
          />
        </a>
      </div>
    </footer>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
